react/week-1/lesson.md

```diff
@@ -59,47 +59,23 @@ Let's take a look at an example. We are going to walk through how to render a `<
 
 First, lets recap how we could do this using "vanilla" JS ([interactive version](https://jsbin.com/zexiyulavu/edit?html,output)):
 
-```html
-<!DOCTYPE html>
-<html>
-<body>
-<div id="root"></div>
-
-<script type="text/javascript">
-  var divNode = document.createElement('div');
-  divNode.innerText = 'Hello World';
-
-  var rootElement = document.querySelector('#root');
-  rootElement.appendChild(divNode);
-</script>
-</body>
-</html>
+```js
+var divNode = document.createElement("div");
+divNode.innerText = "Hello World";
+
+var rootElement = document.querySelector("#root");
+rootElement.appendChild(divNode);
 ```
 
 Now let's convert to using React ([interactive version](https://jsbin.com/calaqitede/1/edit?html,output)):
 
-```html
-<!DOCTYPE html>
```

Comment:
The original intention was to show that React isn't "magic", as CRA hides away a lot of this stuff.

But on reflection I think I agree with this change. The HTML just adds cognitive load. 👍

Reply:
The JSBin examples still have the complete version 👍